### PR TITLE
Open-source Mural Connector.

### PR DESCRIPTION
**Pull request by dmytroIvanovDev**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Mural/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Mural Connector
```
Comment (sriyen-msft) [inline certified-connectors/Mural/README.md:1]: @dmytroIvanovDev Please go thru the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) file and add required information.

File: certified-connectors/Mural/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1074 @@
...
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "object",
+                  "properties": {
+                    "backgroundColor": {
+                      "type": "string",
+                      "description": "backgroundColor"
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiDefinition.swagger.json:50]: @dmytroIvanovDev All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Mural/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiProperties.json:8]: @dmytroIvanovDev Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

File: certified-connectors/Mural/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
+    "scriptOperations": [
+      "CreateNewStickyNote"
+    ],
+    "capabilities": [],
+    "policyTemplateInstances": [],
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiProperties.json:37]: @dmytroIvanovDev If your connector has no policy templates, please remove this property. Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by dmytroIvanovDev** [comment]
@sriyen-msft Hello! I have made changes. Please check.

File: certified-connectors/Mural/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1074 @@
...
+                    },
+                    "sharingSettings": {
+                      "type": "object",
+                      "properties": {
+                        "link": {
+                          "type": "string",
+                          "description": "link"
+                        }
+                      },
+                      "description": "sharingSettings"
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiDefinition.swagger.json:113]: @dmytroIvanovDev **All the description values of properties in the swagger** are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Mural/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[[ 00000000-00000000-00000000-00000000]]",
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiProperties.json:8]: @dmytroIvanovDev Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
"clientId": "[[DUMMY]]"

File: certified-connectors/Mural/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/refresh"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiProperties.json:32]: @dmytroIvanovDev You have agreed to not have white color as iconBrandColor for your connector icon, but you still have provided white color. Please update it.

image

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by dmytroIvanovDev** [comment]
@sriyen-msft Hey! I have made changes for the iconBrandColor. Please check.

File: certified-connectors/Mural/apiProperties.json (hunk 6 above)
Comment (sriyen-msft) [reply certified-connectors/Mural/apiProperties.json:8]: @dmytroIvanovDev This needs to be addressed and resolved

File: certified-connectors/Mural/apiDefinition.swagger.json (hunk 5 above)
Comment (sriyen-msft) [reply certified-connectors/Mural/apiDefinition.swagger.json:113]: @dmytroIvanovDev This needs to be addressed and resolved.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Mural/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,1074 @@
...
+                      "type": "string",
+                      "description": "status"
+                    },
+                    "thumbnailUrl": {
+                      "type": "string",
+                      "description": "thumbnailUrl"
+                    },
+                    "timerSoundTheme": {
+                      "type": "string",
+                      "description": "timerSoundTheme"
```
Comment (sriyen-msft) [inline certified-connectors/Mural/apiDefinition.swagger.json:125]: @dmytroIvanovDev **All the description values of properties in the swagger** are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**